### PR TITLE
Adding Help/ Question template for issues

### PR DESCRIPTION
**Pull request by veritem**
Regarding #141 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm!

File: .github/ISSUE_TEMPLATE/help.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Help
+about: If you have any questions regarding fastify this is the right place to ask
```
Comment (jsumners) [inline .github/ISSUE_TEMPLATE/help.md:3]:
```suggestion
about: If you have any questions regarding Fastify this is the right place to ask
```

File: .github/ISSUE_TEMPLATE/help.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Help
+labels: help wanted
+assignees: ''
+
+---
+
+<!--
+Before you submit an a question we recommend you first look into our  repo's issue section to see if your question have already been answered, we don't recommend duplicates.
```
Comment (jsumners) [inline .github/ISSUE_TEMPLATE/help.md:11]:
```suggestion
Before you submit a question we recommend you first look at our issue section to see if your question have already been answered.
```

"issue section" should be a link.

File: .github/ISSUE_TEMPLATE/questions---help.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+name: Questions / Help
+about: If you have any questions regarding fastify this is the right place to ask
```
Comment (jsumners) [inline .github/ISSUE_TEMPLATE/questions---help.md:3]:
```suggestion
about: If you have any questions regarding Fastify this is the right place to ask
```

File: .github/ISSUE_TEMPLATE/questions---help.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+title: Help
+labels: help wanted
+assignees: ''
+
+---
+
+<!--
+Before you submit an a question we recommend you first look into our  repo's issue section to see if your question have already been answered, we don't recommend duplicates.
```
Comment (jsumners) [inline .github/ISSUE_TEMPLATE/questions---help.md:11]: Same review comment as in the other file.

**Comment by RafaelGSS** [comment]
Might be good to add a link to `minimal-reproduction` found https://github.com/fastify/.github/issues/5

**Comment by makuzaverite** [comment]
cc @jsumners

**Comment by jsumners** [comment]
> cc @jsumners

My unresolved review items remain.

**Comment by makuzaverite** [comment]
I added an update

**Comment by jsumners** [comment]
In the text, there is the statement "see our issue section". I want to see that provide an actual link to the referenced "issue section" because as it is, it could be any one of about about 50 issue sections.

File: .github/ISSUE_TEMPLATE/help.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ assignees: ''
 ---
 
 <!--
-Before you submit a question we recommend you first look at our issue section to see if your question have already been answered.
+Before you submit a question we recommend you first look at our issue section here https://github.com/fastify/help/issues  to see if your question have already been answered.
```
Comment (jsumners) [inline .github/ISSUE_TEMPLATE/help.md:11]:
```suggestion
Before you submit a question we recommend you first look at our issue section (https://github.com/fastify/help/issues) to see if your question has already been answered.
```

**Review by jsumners — APPROVED** [review]
LGTM

**Pull request merged**